samples/cpp/visual_language_chat/visual_language_chat.cpp

```diff
@@ -3,17 +3,22 @@
 
 #include "load_image.hpp"
 #include <openvino/genai/visual_language/pipeline.hpp>
+#include <filesystem>
 #include <openvino/runtime/intel_gpu/properties.hpp>
 
+namespace fs = std::filesystem;
```

Comment:
`fs` and <filesystem> is not used in current sample

Reply:
removed